### PR TITLE
052020 updates

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_shogun/sortmerna/__init__.py [hunk 1]
```diff
@@ -23,6 +23,17 @@
 outputs = {'Non-ribosomal reads': 'per_sample_FASTQ',
            'Ribosomal reads': 'per_sample_FASTQ'}
 
+# defining default parameter set AKA what's going to be shown to the user
+# as options for the command
+dflt_param_set = {
+    'Ribosomal read filtering': {
+        'Output blast format': 1,
```
Comment (ElDeveloper) [inline qp_shogun/sortmerna/__init__.py:30]: Quick question, the type of the arguments here is expressed as an integer, and as a string in the `opt_params` declaration. Is that right?

**Review by ElDeveloper — APPROVED** [review]
One question, otherwise everything looks good.

File: qp_shogun/sortmerna/__init__.py (hunk 1 above)
Comment (antgonza) [reply qp_shogun/sortmerna/__init__.py:30]: As far as I remember, that is correct.

**Comment by ElDeveloper** [comment]
Thanks @antgonza!

**Pull request merged**